tests/io/mongodb/test_cli.py

```diff
@@ -29,7 +33,28 @@ def test_version():
     assert exitcode == 0
 
 
-def test_mongodb_load_table(caplog, cratedb, mongodb):
+DOCUMENT_IN = {
+    "id": bson.Binary.from_uuid(UUID("d575540f-759d-4653-a4c4-4a9e410f1aa1")),
+    "value": {
+        "name": "foobar",
+        "active": True,
+        "created": dateutil.parser.parse("2020-06-19T15:03:53.727Z"),
+        "timestamp": bson.datetime_ms.DatetimeMS(1455141600000),
+    },
+}
+DOCUMENT_OUT = {
+    "__id": mock.ANY,
+    "id": "d575540f-759d-4653-a4c4-4a9e410f1aa1",
+    "value": {
+        "name": "foobar",
+        "active": True,
+        "created": 1592579033000,
+        "timestamp": 1455141600000,
+    },
+}
```

Comment:
Best view into the integration test added with bf78869b3f943b67b0c0c06e27a00fe54b0c560a.